CHANGELOG.md

```diff
@@ -1,5 +1,84 @@
+* [maestro] Updated maestro to [v2.9.0](https://github.com/PelionIoT/maestro/releases/tag/v2.9.0)
+  * Gateway capabilities - Allows gateways to advertise the features supported by the platform. Maestro utilizes Edge Core's GRM JSON-RPC APIs to add to the gateway device's LwM2M resources. The registered resources are added under Pelion's reserved FeatureMgmt LwM2M object - 33457 with 3 resources - 0 - featureID, 1 - enabled, 2 - config.
+  * Remote config management via LwM2M - FeatureMgmt config resource allows users to remotely view the current configuration of the feature and also push a config update using the LwM2M cloud service APIs. Maestro, on receiving an update, writes the content to the file path specified in the respective parameter config_filepath.
+  * Removed parsing and generation of self-signed certificates. Also removed the platforms rp200 and wwrelay which are no longer supported.
+  * Maestro traffic now routed through edge-proxy instead of directly connecting to the Pelion Cloud.
+  * By default, disabled the maestro's gateway logging feature in the configuration file for platforms - `imx8mmevk`, `uz3eg-iocc` and `raspberrypi3`. Gateway logging feature is supported through fluentbit.
+* [logrotate] Added bbappend file to install a configuration which rotates the logs of files under `/var/log`.
+* [logrotate] Removed crontab method to call logrotate every 5 mins. Instead configured systemd logrotate timer to 5 mins.
+* [identity-tool] Established an independent recipe under recipes-connectivity and removed the dependency on wwrelay-utils.
+  * Converted the identity node program to bash which now lives in a new project called pe-utils.
+  * Installed v2.0.4 of identity-tool, which majorly removes the generation of self-signed certificate.
+* [edge-proxy] Configured services gateway logs (maestro), gateway stats (maestro), relay-term, fluentbit http endpoint to route traffic through edge-proxy and not consume self-signed certificate.
+* [meta-nodejs] Removed the dependency on node v8.x. Upgraded the node packages to work with default nodejs version of Dunfell.
+* [relay-term] Upgraded relay-term to work with node v12.x. Established an independent recipe from wwrelay-utils and removed the dependency on global-node-modules.
+* [mbed-fcce] Upgraded factory-configurator-client-example to v4.7.0 and pinned its respective dependencies.
+* [fluentbit] Added recipe to install Fluentbit 1.3.5 on the gateway for providing an open source Log Processor and Forwarder solution.
+  * By default, fluentbit is configured with following input endpoints - CPU, MEM, Systemd services - edge-core, edge-proxy, pelion-relay-term, maestro, kubelet, docker and wait-for-pelion-identity. And an output endpoint HTTP to publish logs at API endpoint - `http://gateways.local:8080/v3/device-logs` (routing through edge-proxy).
```

Comment:
Could you please rephrase this bullet? I'm not sure what we're trying to say.

Reply:
That is a bit technical. Someone who is familiar with Fluent Bit will get that. Should we simplify it?